### PR TITLE
torch concat backward compatibility

### PR DESCRIPTION
**Pull request by mojtaba-komeili**
**Patch description**
Backward compatibility for `torch.concat` function in cringe loss.

### CODE REVIEW

File: projects/cringe/cringe_loss.py [hunk 1]
```diff
@@ -88,9 +88,15 @@ def __call__(self, x, y, classifier_labels=None, **kwargs):
...
-        )
+        if hasattr(torch, 'concat'):
+            # torch > 1.10
+            x_ct = torch.concat(
+                [x_target.unsqueeze(1), sample_preds_values.unsqueeze(1)], -1
+            )
+        else:
+            x_ct = torch.cat(
+                [x_target.unsqueeze(1), sample_preds_values.unsqueeze(1)], -1
+            )
```
Comment (klshuster) [inline projects/cringe/cringe_loss.py:99]: we could just do `torch.cat` to avoid the check? pretty sure it's just an alias

**Review by stephenroller — APPROVED** [review]

**Review by klshuster — APPROVED** [review]
